### ISSUE TITLE
Update Limit Clause Syntax

### ISSUE THREAD

**Issue by alexsasharegan** [FaaPz/PDO#78]
When running an update with a limit clause, Slim PDO uses the normal MySQL LIMIT syntax for SELECTS `[LIMIT {[offset,] row_count | row_count OFFSET offset}]`. This is not correct for UPDATE statements. See the excerpt from [the MySQL 5.7 docs](https://dev.mysql.com/doc/refman/5.7/en/update.html):

```sql
UPDATE [LOW_PRIORITY] [IGNORE] table_reference
    SET col_name1={expr1|DEFAULT} [, col_name2={expr2|DEFAULT}] ...
    [WHERE where_condition]
    [ORDER BY ...]
    [LIMIT row_count]
```

If I've read through the issues correctly, there is already a fix coming for the `offset` integer error.

**Comment by WFrancois** [comment]
(Continuing on this thread)

@alexsasharegan Well now it should work, as with the fix on #89 it doesn't add the OFFSET anymore if it isn't set ?

**Comment by alexsasharegan** [comment]
Ooh, good point! I'll close this and reopen if any issues arise.
